README.md

```diff
@@ -341,6 +341,22 @@ The query layout is intended for search, filter, and result pages.The query layo
 └————————————————————————————┘
 ```
 
+To make the main content and heading area larger on desktop, delete the aside sidebar element `o-layout__aside-sidebar`.
```

Comment:
Could we say something more along the line of 'this layout can be used without a right-hand side bar, which you will get by delelting the aside ....'